test/components/practice.spec.coffee

```diff
@@ -0,0 +1,148 @@
+{expect} = require 'chai'
+_ = require 'underscore'
+React = require 'react/addons'
+Router = require 'react-router'
+Promise = require('es6-promise').Promise
+
+{CourseActions, CourseStore} = require '../../src/flux/course'
+{TaskActions, TaskStore} = require '../../src/flux/task'
+{TaskStepActions, TaskStepStore} = require '../../src/flux/task-step'
+
+{SinglePractice, Tasks} = require '../../src/components'
+{routes} = require '../../src/router'
+
+VALID_MODEL = require '../../api/courses/1/practice.json'
+
+
+div = document.createElement('div')
+
+routerHelper = (route) ->
+  history = new Router.TestLocation([route])
+
+
+  promise = new Promise (resolve, reject) ->
+    Router.run routes, history, (Handler, state)->
+      router = @
+      React.render(<Handler/>, div, ()->
+        component = @
+        result = {div, component, state, router, history}
+        resolve(result)
+      )
+
+tasksHelper = (courseId) ->
+  routerHelper('/courses/' + courseId + '/tasks/')
```

Comment:
You can use coffeescript's String concatenation syntax to make this a bit more concise: `"/courses/#{courseId}/tasks"` (is the trailing slash necessary?)